### DISCUSSION TITLE
Uploading large number of fingerprints fails

### DISCUSSION THREAD

**Discussion by ninjamonkey198206** [stashapp/stash#3535] [Q&A]
Just wondering if anyone else has gone through and matched like, I don't know, 900+ scenes and then realized that they had forgotten to upload the fingerprints?

I did that, and now the upload fails.

Anyone have any ideas about why or how to get them pushed?

**Comment by DogmaDragon** [comment]
The only time I know someone encountered a failure to submit fingerprints was if they had one queued up, but then deleted the scene that used that fingerprint. 

Do you get any errors when clicking submit in the browser console (Ctrl+Shift+I)?

**Reply by ninjamonkey198206** [reply]
```text
Failed to load resource: the server responded with a status of 504 (Gateway Time-out)

Unexpected token '<', "<html><bod"... is not valid JSON               index.cfbc3cfa.js:153 
error @ index.cfbc3cfa.js:153
te @ SceneList.0f8f901c.js:1
```
I had previously uploaded several batches, and testing the credentials works.

**Reply by ninjamonkey198206** [reply]
Wait...
Ran a test cleaning and found some "missing" files. Running a proper cleaning now.

**Reply by ninjamonkey198206** [reply]
Well, that seems to have cleared out the erroneous scenes, but the fingerprint count is the same and the upload still fails.
Unsure of what to do now.

**Reply by DogmaDragon** [reply]
The clean task doesn't remove the fingerprints from the queue, so not surprised it didn't fix it.

Not sure what's causing it. It might be exactly what you are suspecting that the queue is too long and it times out. The devs can probably help you with that. 

Fingerprints are stored in the browser's IndexedDB. You can delete the database attached to clear it, but it will remove the queue and tagger preferences. Ctrl+Shift+I > Application > IndexedDB > localforage
